CONTRIBUTING.md

```diff
@@ -499,7 +499,7 @@ coordinate to avoid any conflicts and always warn people beforehand if you are
 to work on their branch. Once you are done:
 
 ```
-git commit -m "your additions"
+git commit -m "Add my contribution\n\nThe PR was missing something. I added it."
```

Comment:
This won't work: https://stackoverflow.com/questions/5064563/add-line-break-to-git-commit-m-from-the-command-line

Use any of the suggestions provided in the SO answer I linked.

i.e.

```bash
git commit -m 'Add my contribution

The PR was missing something. I added it.'
```

Reply:
thanks for the catch! changed it  :)